src/Resources/contao/modules/ModuleArticle.php

```diff
@@ -366,7 +366,7 @@ public function generatePdf()
 		// Close and output PDF document
 		$pdf->lastPage();
 		$pdf->Output(standardize(ampersand($this->title, false)) . '.pdf', 'D');
-
+		// FIXME: This has to be changed to a ResponseException somehow - rather write to temp file and stream it then?
```

Comment:
We should use the `S` option as second parameter which returns the value. Probably should send it to a stream and use a stream response. I have never used streams though ;-)